state-manage-assign/src/components/Table.tsx

```diff
@@ -0,0 +1,260 @@
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  
+  
+ 
+  return (
+    <div>
+      
+      <div className='header-top'>
+        <Text level={'Candidates'}></Text>
+        <div >
+        
+        <Button className='button' variant="outlined" startIcon={<SystemUpdateAltIcon />} 
+          style={{borderColor: '#696A6E', color : '#696A6E', margin : '15px', textTransform:"none"}}
+          
+          >
+          Export
+        </Button>
+        
+        <Button className='button' variant="contained" startIcon={<AddIcon />}
+          style={{ color: '#FFF',borderColor: 'blue', textTransform:"none"}}>
+          Manual Order
+        </Button>
+        
+        </div>
+        
+    </div>
+    <Paper className='parentCont' style={{borderRadius:'20px'}}>
+    <div className='header'>
+        <Text level={'Candidate Information'}></Text>
+        {/* <div> */}
+        <TextField
+          label="Search by Name"
+          value={searchQuery}
+          onChange={(e) => { setSearchQuery(e.target.value); setFlag((prevFlag )=> !prevFlag);}}
+          sx = {{width : '30%', height : '25px' }}
+        />
+        
+        <div>
+        <Button
+          id="demo-positioned-button"
+          aria-controls={open ? 'demo-positioned-menu' : undefined}
+          aria-haspopup="true"
+          aria-expanded={open ? 'true' : undefined}
+          onClick={handleClick}
+        >
+          <Button variant="outlined" startIcon={<FilterListIcon />} 
+            style={{borderColor: '#E5E7ED', color : 'black', margin : '10px'}}
+            >
+              Filter</Button>
+          
+        </Button>
+        <Menu
+          id="demo-positioned-menu"
+          anchorEl={anchorEl}
+          open={Boolean(anchorEl)}
+          onClose={handleClose}
+          onKeyDown={handleListKeyDown}
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'right',
+          }}
+          
+        > 
+          <MenuItem> Filters </MenuItem>
+          <hr></hr>
+        
+          <MenuItem> Status </MenuItem>
+          <MenuItem onClick={() => setStat('All')}>
+            <Checkbox checked={stat === 'All'} />All
+          </MenuItem>
+          <MenuItem onClick={() => setStat('CLEAR')}>
+            <Checkbox checked={stat === 'CLEAR'} />Clear
+          </MenuItem>
+          <MenuItem onClick={() => setStat('CONSIDER')}>
+            <Checkbox checked={stat === 'CONSIDER'} />Consider
+          </MenuItem>
+          <MenuItem> Ajudication </MenuItem>
+          <MenuItem onClick={() => setStat('All2')}>
+            <Checkbox checked={stat === 'All2'} />All
+          </MenuItem>
+          <MenuItem onClick={() => setStat('Engaged')}>
+            <Checkbox checked={stat === 'Engaged'} />Engaged
+          </MenuItem>
+          <MenuItem onClick={() => setStat('Pre adverse action')}>
+            <Checkbox checked={stat === 'Pre adverse action'} />Pre adverse action
```

Comment:
Don't hardocde these actions in jsx
move to a constants.tsx file and import from there